drivers/SmartThings/matter-evse/src/test/test_evse_energy_meas.lua

```diff
@@ -0,0 +1,215 @@
+  local subscribe_request = cluster_subscribe_list[1]:subscribe(mock_device)
+  for i, cluster in ipairs(cluster_subscribe_list) do
+    if i > 1 then
+      subscribe_request:merge(cluster:subscribe(mock_device))
+    end
+  end
+  test.socket.matter:__expect_send({ mock_device.id, subscribe_request })
+  test.mock_device.add_test_device(mock_device)
+  test.socket.device_lifecycle:__queue_receive({ mock_device.id, "added" })
+
+  test.socket.capability:__expect_send(mock_device:generate_test_message("main",
+  capabilities.evseChargingSession.targetEndTime("1970-01-01T00:00:00Z")))
+end
+test.set_test_init_function(test_init)
+
+test.register_coroutine_test(
+  "Assert profile applied over doConfigure",
+  function()
+    test.socket.device_lifecycle:__queue_receive({ mock_device.id, "doConfigure" })
+    mock_device:expect_metadata_update({ profile = "evse-energy-meas" })
+    mock_device:expect_metadata_update({ provisioning_state = "PROVISIONED" })
+  end
+)
+
+test.register_coroutine_test(
+  "Ensure timers are created for the device",
+  function()
+    local poll_timer = mock_device:get_field("__recurring_poll_timer")
+    assert(poll_timer ~= nil, "poll_timer should exist")
+
+    local report_poll_timer = mock_device:get_field("__recurring_report_poll_timer")
+    assert(report_poll_timer ~= nil, "report_poll_timer should exist")
+  end
+)
+
+test.register_coroutine_test(
+  "Ensure timers are created for the device",
+  function()
+    test.socket.matter:__set_channel_ordering("relaxed")
+
+    test.socket.device_lifecycle:__queue_receive({ mock_device.id, "removed" })
+    test.wait_for_events()
+
+    local poll_timer = mock_device:get_field("__recurring_poll_timer")
+    assert(poll_timer == nil, "poll_timer should not exist")
+
+    local report_poll_timer = mock_device:get_field("__recurring_report_poll_timer")
+    assert(report_poll_timer == nil, "report_poll_timer should not exist")
+  end
+)
+
+test.register_coroutine_test(
+  "Ensure that every 60 seconds the driver reads the CumulativeEnergyImported attribute for both endpoints",
+  function()
+    test.mock_time.advance_time(60)
+    test.socket.matter:__set_channel_ordering("relaxed")
+    local CumulativeEnergyImportedReadReq = clusters.ElectricalEnergyMeasurement.attributes.CumulativeEnergyImported
+        :read(mock_device, ELECTRICAL_SENSOR_EP_ONE)
+    CumulativeEnergyImportedReadReq:merge(clusters.ElectricalEnergyMeasurement.attributes.CumulativeEnergyImported:read(
+      mock_device, ELECTRICAL_SENSOR_EP_TWO))
+    test.socket.matter:__expect_send({
+      mock_device.id,
+      CumulativeEnergyImportedReadReq
+    })
+    test.wait_for_events()
+  end,
+  {
+    test_init = function()
+      test_init()
+      test.timer.__create_and_queue_test_time_advance_timer(60, "interval", "create_poll_schedule")
+    end
+  }
+)
+
+test.register_coroutine_test(
+  "Ensure the total accumulated powerConsumption for both endpoins is reported every 15 minutes",
```

Comment:
```suggestion
  "Ensure the total accumulated powerConsumption for both endpoints is reported every 15 minutes",
```

This test also failed in the latest run

Reply:
Yes, I fixed this intermittently failing test and the typo the other day, but Hunsup pushed some changes that reverted this. It will be fixed in the next commit.